gf-webapp/src/component/pageBorder/header/burger/Burger.tsx

```diff
@@ -0,0 +1,37 @@
+import {useState} from "react";
+import {useScrollPosition} from "src/utils/domEventsUtils/useScrollPosition";
+import styles from "src/component/pageBorder/header/burger/Burger.module.scss";
+import {BurgerContent} from "./burgerContent/BurgerContent";
+
+export const Burger = () => {
+  const [burgerActive, setBurgerActive] = useState(false);
+
+  const burgerOpenHandler = () => {
+    setBurgerActive(prev => !prev);
+    document.body.classList.toggle("notScrollable");
+  };
+
+  const scrollPosition = useScrollPosition();
```

Comment:
It may be better to move the scrollPosition declaration right after the 7 line or before it.
It may be worthwhile to look up such points everywhere. And if possible, declare variables first and functions after them

Reply:
Done